src/components/PaymetsTable/PaymentsTable.js

```diff
@@ -1,10 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import CustomTable from '../UI/CustomTable/CustomTable';
 import { IconButton } from '@material-ui/core';
 import EditIcon from '@material-ui/icons/Edit';
-import { Link } from 'react-router-dom';
 import { format } from 'date-fns';
+import EditPaymentForm from './../../containers/Forms/Payment/EditPaymentForm';
+
 const PaymentsTable = ({ paymentsData }) => {
+  const [isOpen, setIsOpen] = useState({ status: false });
```

Comment:
Вроде объект лишний здесь
```suggestion
  const [isOpen, setIsOpen] = useState(false);
```

Reply:
без этого не работает передача открытия окна, если просто оставить false, 2-й раз не открывается.